specs/lib/cacheOptimizeIndicesSpec.js

```diff
@@ -5,6 +5,7 @@ var readGltf = require('../../lib/readGltf');
 var readAccessor = require('../../lib/readAccessor');
 var writeAccessor = require('../../lib/writeAccessor');
 
+var basePath = './specs/data/boxTexturedUnoptimized/';
```

Comment:
There are a few lines like this that aren't used. Otherwise this PR looks ready to me.